cmd/utils.go

```diff
@@ -23,6 +23,10 @@ func loadDev(devPath string) (*model.Dev, error) {
 }
 
 func askYesNo(q string) bool {
+	if len(os.Getenv("OKTETO_YES")) > 0 {
```

Comment:
`OKTETO_ASK_FOR_DEVENV_CREATION_CONFIRMATION`?

Reply:
that's way too long even for a copy paste. Maybe  `OKTETO_SILENT` or `OKTETO_AUTODEPLOY` ?